packages/connect-examples/electron-example/src/index.ts

```
@@ -1,18 +1,10 @@
-import {
-  screen,
-  app,
-  BrowserWindow,
-  session,
-  ipcMain,
-  USBDevice,
-  SerialPort,
-  HIDDevice,
-} from 'electron';
+import { screen, app, BrowserWindow, session, ipcMain } from 'electron';
 import path from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)